src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue889.cs

```diff
@@ -12,18 +12,16 @@ public Issue889(TestDevice testDevice) : base(testDevice)
 
 	public override string Issue => "Assigning to FlyoutPage.Detail after construction doesn't work";
 
-	//[Test]
-	//[Category(UITestCategories.FlyoutPage)]
-	//[Description("Reproduce app crash - Issue #983")]
-	//public void Issue899TestsAppCrashWhenSwitchingTabs()
-	//{
-	//	App.Tap(q => q.Marked("Push new page"));
-	//	App.WaitForElement(q => q.Marked("I have been pushed"));
-	//	App.Screenshot("Push page");
-	//	App.Back();
-	//	App.Screenshot("Navigate back");
+	[Test]
+	[Category(UITestCategories.FlyoutPage)]
+	public void Issue899TestsAppCrashWhenSwitchingTabs()
```

Comment:
Failing iOS and Catalyst:
<img width="339" alt="image" src="https://github.com/user-attachments/assets/ee68a248-aecf-4e14-bec3-ef1e72ed5bd8">

Reply:
@jsuarezruiz  Updated the tests and committed the changes. Kindly review and share your concerns.
<img width="442" alt="image" src="https://github.com/user-attachments/assets/c5ebd6ca-72f6-4622-902b-e22b2305677d">